fdbserver/DDTeamCollection.actor.cpp

```diff
@@ -3328,15 +3328,19 @@ void DDTeamCollection::updateCpuPivots() {
 }
 
 void DDTeamCollection::updateTeamEligibility() {
+	int healthyCount = 0, lowDiskCount = 0, lowCpuCount = 0;
 	for (auto& team : teams) {
 		if (team->isHealthy()) {
+			healthyCount++;
 			if (team->hasHealthyAvailableSpace(teamPivots.pivotAvailableSpaceRatio)) {
+				lowDiskCount++;
```

Comment:
`lowDiskCount` can sound misleading as it might mean that the amount of disk available is low, which is the opposite of what we want to convey. This can instead be called `lowDiskUtilCount`. (Same for the trace event field name.)